src/components/PlioQuestion.vue

```diff
@@ -127,6 +147,11 @@ export default {
       isAnswerCorrect: false,
       isAnswerSubmitted: false,
       showButtonLoading: false,
+      updatedProgressBarInfo: {
+        "config": this.progressBarInfo['config'],
+        "updatedCompletionPercent": this.progressBarInfo['completionPercent'],
```

Comment:
Instead of `updated` -- rename to `newCompletionPercent`. This is for the animation from old position to new position no?